src/person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age:, name: 'unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age? ? true : @parent_permission
+  end
+
```

Comment:
- [ ] I think we can save some code here.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```

Reply:
🔥 Fixed